### PR TITLE
fix(Dashboard): fix space between footer and empty message

### PR DESCRIPTION
**Pull request by jyenes**
####What does this PR do?
fix space between footer and empty message
closes: #908

### CODE REVIEW

File: app/components/pages/Dashboard/index.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ const MobileOnlyStickyFooter = styled(StickyFooter)`
 `
 
 const CenterdSmallParagraph = styled(SmallParagraph)`
+  margin-top: ${th('space.7')};
```
Comment (jsms90) [inline app/components/pages/Dashboard/index.js]: This spacing (space.7) is much larger than it used to be (space.2). It has this affect on the submissions tab:
image

Was this intentional?

Also, please see the last PR comment about `margin-top`. This issue is probably best resolved by applying a `min-height` to the `TabbedSubmissions` component & leaving the existing `margin-bottom` (unless it needs to be increased to `space.7`?)

Comment (jyenes) [reply app/components/pages/Dashboard/index.js]: @jsms90 sorry, my bad, I will fix it now

Comment (jsms90) [reply app/components/pages/Dashboard/index.js]: no problem at all :+1:

**Comment by chugginselifesciences** [comment]
I feel like it needs more space when there are enough items in the list so it's top edge isn't so close. But looks ok when empty/in archive. No problem if it needs to go a bit lower to account for extra space when the list is full enough that the min-width isn't a factor.

**Comment by jsms90** [comment]
@chugginselifesciences how much space would you like?

**Comment by chugginselifesciences** [comment]
@jsms90 it looks like it's around 8px from the bottom of the list at the moment. Without mocking it up I would guess that the 'standard' vertical space of 24px would be enough. Thanks.

File: app/components/pages/Dashboard/index.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ const BoxHiddenFromMobile = styled(Box)`
 `
 
 const TabbedSubmissions = styled(Tabs)`
+  min-height: 300px;
   margin-bottom: ${th('space.2')};
```
Comment (jsms90) [inline app/components/pages/Dashboard/index.js]: Which means that this just needs to be `space.3` @jyenes

Comment (jyenes) [reply app/components/pages/Dashboard/index.js]: done

**Comment by diversemix** [comment]
New screenshots .... @chugginselifesciences 

image
-----
and....
image

**Pull request merged**